fe1-web/__tests__/model/network/FromJson.ts

```diff
@@ -38,6 +39,15 @@ const _generateKeyPair = () => {
 
 describe('=== fromJson object checks ===', function() {
 
+  beforeAll(() => {
+    const promise = new Promise((resolve, reject) => {
+      initialise();
+    });
+    promise.then(
+      () => console.log('storage configured (printed from storage setup)'),
+    );
+  });
```

Comment:
That is a very strange point. I also googled yesterday to learn how to handle promises in async code in jest. I did exactly what you (and Jest) proposed. However, doing so, for some reason, does the polar inverse of what we expect. Returning a promise makes it so that the tests don't wait for `beforeAll()` to complete before executing (thus resulting is fails because nothing is in the storage)